nav2_smac_planner/include/nav2_smac_planner/node_lattice.hpp

```diff
@@ -40,7 +41,37 @@ namespace nav2_smac_planner
 class NodeLattice;
 class NodeHybrid;
 
-typedef std::pair<unsigned int, double> LatticeMetadata;
+/**
+ * @struct nav2_smac_planner::LatticeMetadata
+ * @brief A struct of all lattice metadata
+ */
+struct LatticeMetadata
+{
+  float turningRadius;
```

Comment:
Can we make this `min_turning_radius` to be clear?

Reply:
Done